plugins/content-manager/src/main/java/com/wazuh/contentmanager/cti/console/service/AuthServiceImpl.java

```diff
@@ -30,16 +31,14 @@ public AuthServiceImpl() {
 
     /**
      * Obtains a permanent token for the instance from CTI Console.
-     * @param clientId unique client identifier for the instance.
-     * @param deviceCode unique device code provided by the CTI Console during the registration of the instance.
+     * @param s registration details of the instance.
      * @return access token.
      */
-    // TODO replace parameters with SubscriptionModel from https://github.com/wazuh/wazuh-indexer-plugins/pull/662
     @Override
-    public Token getToken(String clientId, String deviceCode) {
+    public Token getToken(Subscription s) {
         try {
             // Perform request
-            SimpleHttpResponse response = this.client.getToken(clientId, deviceCode);
+            SimpleHttpResponse response = this.client.getToken(s.getClientId(), s.getDeviceCode());
```

Comment:
Use more descriptive names

```suggestion
    public Token getToken(Subscription subscription) {
try {
// Perform request
            SimpleHttpResponse response = this.client.getToken(subscription.getClientId(), subscription.getDeviceCode());
```

Reply:
Applied in 2728ad222e36c2e5a7614ccd81a4741a9b3404e1